src/content/docs/en/guides/media/cloudinary.mdx

```diff
@@ -6,11 +6,214 @@ stub: true
+
+## Loading Images & Videos
+
+The Astro Content Layer can be used to load Cloudinary assets with the `cldAssetsLoader`.
+
+To load images or videos, add:
+
+```js
+import { defineCollection } from 'astro:content';
+import { cldAssetsLoader } from 'astro-cloudinary/loaders';
+
+export const collections = {
+  assets: defineCollection({
+    loader: cldAssetsLoader({
+      folder: '<Folder>' // Optional, without loads root directory
+    })
+  }),
+}
+```
+
+You can then use the [getCollection or getEntry functions](/en/guides/content-collections/#querying-collections) to select one or many images or videos from your collection.
+
+[Learn more about cldAssetsLoader](https://astro.cloudinary.dev/cldassetsloader/basic-usage).
+
+## Generating Social Media Cards
+
+Social Media Cards are useful for increasing engagement on social posts.
+
+To create a Social card, generate a Cloudinary URL and utilize Open Graph and Twitter meta tags to embed the image on your website:
+
+```jsx
+---
+import { getCldOgImageUrl } from 'astro-cloudinary/helpers';
+const ogImageUrl = getCldOgImageUrl({ src: '<Public ID>' });
+---
+<meta property="og:image" content={ogImageUrl} />
+<meta property="og:image:secure_url" content={ogImageUrl} />
+<meta property="og:image:width" content="1200" />
+<meta property="og:image:height" content="630" />
+<meta property="twitter:title" content="<Twitter Title>" />
+<meta property="twitter:card" content="summary_large_image" />
+<meta property="twitter:image" content={ogImageUrl} />
+```
+
+The `getCldOgImageUrl` helper utilizes the same API as the CldImage component, meaning you can use Cloudinary transformations to create dynamic, unique social cards for any of your pages.
+
+Find [Social Media Card templates](https://astro.cloudinary.dev/templates/social-media-cards) on the Astro Cloudinary docs.
+
+[Learn more about getCldOgImageUrl](https://astro.cloudinary.dev/getcldogimageurl/basic-usage).
+
+## Using Cloudinary in Node.js
+
+For more complex asset management, uploading, or analysis, you can use the Cloudinary Node.js SDK when working in an Astro Node.js environment.
+
+Install the Cloudinary Node.js SDK by running:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install cloudinary
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm install cloudinary
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add cloudinary
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+And configure your account by running:
+
+```js
+import { v2 as cloudinary } from "cloudinary";
+
+cloudinary.config({
+  cloud_name: import.meta.env.PUBLIC_CLOUDINARY_CLOUD_NAME,
+  api_key: import.meta.env.PUBLIC_CLOUDINARY_API_KEY,
+  api_secret: import.meta.env.CLOUDINARY_API_SECRET,
+});
+```
+
+:::note
+This assumes you have your environment variables set up using the specified names.
+:::
+
+Which will give you access to the Cloudinary APIs such as the Upload API, where you can upload an asset by running:
```

Comment:
```suggestion
This will give you access to all of the Cloudinary APIs to allow you to interact with your images.

For example, the Upload API allows you to upload an asset using JavaScript in your Astro project, such as in a client-side `<script>` tag.
```
Ah, I see where "running" came from! In a set of instructions, "running" is always going to mean "run this command now" not "write this function that will run a command".

So, I'm not sure if this is entirely correct, but I think something of the structure above would be a helpful way to describe this.

Reply:
yeah 🙈  addressed in latest